src/lib/constant.js

```diff
@@ -0,0 +1,2 @@
+export const MONITOR_ENABLED_MSG = 'Extensions are being monitored';
```

Comment:
Why have these strings become constants? This introduces some inconsistency: lots of UI strings are embedded in the code itself, except for some strings.

It looks like this was used in tests; in that case you could keep the original implementation as is, and declare constants in the test with a comment before pointing out that this is a string in ext-popup.js